### DISCUSSION TITLE
solidity可以用于做什么嘞？

### DISCUSSION THREAD

**Discussion by ChenHaoJie9527** [AmazingAng/WTF-Solidity#414] [Q&A] [answered]
从事前端开发，因业务发展，接触到智能合约，想知道Soldity这门语言可以从事什么工作嘞？
学习solidity，对我目前的职业发展有没有实际上的帮助嘞？

**Comment by AmazingAng** [comment, marked as answer]
1. 学好Solidity后可以从事智能合约开发，审计工作，也可以从dapp开发，前端等工作。
2. 如果你前端工作需要与智能合约交互，学会Solidity可以直接帮助到你更好的理解智能合约。

**Reply by ChenHaoJie9527** [reply]
但似乎Solidity安全性比较令人堪忧呀，是否会有其他语言可以代替呢？比如Move语言
